Update issue link from swift to swift-docc (#153)
apple/swift-org-website: documentation-workgroup/index.md

* Discussing ideas in the Swift Forums, for example in the [Swift-DocC
  forum](https://forums.swift.org/c/development/swift-docc) for
  Swift-DocC–related ideas

> * Opening issues to track enhancements and bugs for the Documentation Workgroup’s projects:

Reviewer: ```suggestion
* Opening issues to track enhancements and bugs for the projects governed by the Documentation Workgroup:
```

This might be better, as the Documentation Workgroup is not necessarily the owner of the projects.

>   [swift-docc](https://github.com/apple/swift-docc/issues), [swift-book](https://github.com/apple/swift-book/issues)

Reviewer: Sorry, I just realized that the links can be improved.

I would say either `[repo name](repo link)`:

```suggestion
  [apple/swift-docc](https://github.com/apple/swift-docc), [apple/swift-book](https://github.com/apple/swift-book)
```

Or `[project name](repo issues link)`:

```suggestion
  [Swift-DocC](https://github.com/apple/swift-docc/issues), [The Swift Programming Language book](https://github.com/apple/swift-book/issues)
```

What do you think?

>   [swift-docc](https://github.com/apple/swift-docc/issues), [swift-book](https://github.com/apple/swift-book/issues) and so on

Reviewer: ```suggestion
  [swift-docc](https://github.com/apple/swift-docc/issues), [swift-book](https://github.com/apple/swift-book/issues)
```

I would probably leave out the "and so on" and make sure the list is up-to-date, as to not make readers wonder what other projects there might be (and then not find any).
